### PR TITLE
[SRVKS-779] Add dashboard for knative service queue proxy metrics

### PR DESCRIPTION
**Pull request by skonto**
As per titile.

To test this a service monitor needs to be deployed for the corresponding service. This PR requires docs. 

Screenshots on OCP 4.8:
image
image
image

### CODE REVIEW

File: knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-service-queue-proxy.yaml [hunk 1]
```diff
@@ -0,0 +1,638 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dash-knative-serving-service-queue-proxy
```
Comment (nak3) [inline knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-service-queue-proxy.yaml:4]: This ConfigMap name should be consistent with the ConfigMap (`grafana-dashboard-definition-knative-serving-resources`):
https://github.com/openshift-knative/serverless-operator/blob/fe8d29ddb5d34cd47c352d476f169dcc38c5aa72/knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-resources.yaml#L18

Probably `grafana-dashboard-definition-queue-proxy` or something?

File: knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-service-queue-proxy.yaml [hunk 2]
```diff
@@ -0,0 +1,638 @@
...
+  namespace: openshift-config-managed
+  labels:
+    console.openshift.io/dashboard: "true"
+data:
+  eventing-source-dashboard.json: |+
```
Comment (nak3) [inline knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-service-queue-proxy.yaml:9]: `eventing-source-dashboard.json` should be changed?

**Review by nak3 — COMMENTED** [review]
Left two small comments about naming.

File: knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-service-queue-proxy.yaml (hunk 2 above)
Comment (skonto) [reply knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-service-queue-proxy.yaml]: Something went wrong when pushing the file renaming thing. I changed it.

File: knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-service-queue-proxy.yaml (hunk 1 above)
Comment (skonto) [reply knative-operator/deploy/resources/dashboards/serving/grafana-dash-knative-serving-service-queue-proxy.yaml]: Changed it to: `grafana-dash-definition-queue-proxy`.

**Review by nak3 — APPROVED** [review]
/lgtm
/approve

I think I don't have to approve, though.

**Comment by skonto** [comment]
@rhuss could you take approve this one?

**Pull request merged**